data/src/main/java/io/gnosis/data/models/Owner.kt

```diff
@@ -48,6 +48,14 @@ data class Owner(
                 else -> IMPORTED
             }
         }
+
+        override fun toString(): String =
+            when(value) {
+                0 -> "Imported"
```

Comment:
we should use localized strings from strings.xml instead of hardcoded strings when they are displayed on screen